easybuild/easyconfigs/b/boost_histogram/boost_histogram-1.2.1-foss-2021a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'boost_histogram'
+version = '1.2.1'
+
+homepage = 'https://boost-histogram.readthedocs.io'
+description = """Boost-histogram is a Python package providing Python bindings for Boost.Histogram."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('pybind11', '2.6.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+download_dep_fail = True
+
+exts_list = [
+    (name, version, {
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
```

Comment:
@ekaterinaasf When you use `PythonPackage`, you need to make sure that `sources` is specified too, so you should use this (and ideally right below the `toolchain` line, to stick to the usual "code" style in easyconfigs):

```python
sources = ['%(namelower)s-%(version)s.tar.gz']
checksums = [' a27842b2f1cfecc509382da2b25b03056354696482b38ec3c0220af0fc9b7579']
```

or, equivalently, using a template constant:

```python
sources = [SOURCE_TAR_GZ]
checksums = [' a27842b2f1cfecc509382da2b25b03056354696482b38ec3c0220af0fc9b7579']

Reply:
Changed and pushed, cmd said that it's done, but I cannot see here this new merge.
Could you check when you have a time? Pushed just now.

Reply:
The changes are there, see https://github.com/easybuilders/easybuild-easyconfigs/pull/14516/files